tests/test_midas_open.py

```diff
@@ -6,7 +6,7 @@
 
 
```

Comment:
The broken tests issues has been fixed in this [PR](https://github.com/opencdms/opencdms-test-data/pull/36/files). It also reflects the recent changes in opencdms package.